applicationinsights-react-native/package.json

```diff
@@ -1,97 +1,99 @@
 {
-    "name": "@microsoft/applicationinsights-react-native",
-    "version": "4.3.2",
-    "description": "Microsoft Application Insights React Native Plugin",
-    "main": "dist-esm/index.js",
-    "exports": {
-        ".": {
-            "types": "./types/index.d.ts",
-            "import": "./dist-esm/index.js"
-        },
-        "./manual": {
-            "types": "./types/manualIndex.d.ts",
-            "import": "./dist-esm/manualIndex.js"
-        }
+  "name": "@microsoft/applicationinsights-react-native",
+  "version": "4.3.2",
+  "description": "Microsoft Application Insights React Native Plugin",
+  "main": "dist-esm/index.js",
+  "exports": {
+    ".": {
+      "types": "./types/index.d.ts",
+      "import": "./dist-esm/index.js",
+      "require": "./dist-esm/index.js"
```

Comment:
Wouldn't the "correct" require() import be the bundle we provide that is already wrapped to "handle" require js

ie `./dist/applicationinsights-react-native.js` or `./browser/applicationinsights-react-native.js`

Reply:
Wasn't really sure how that worked, but did a test and that one seems to work too! Will update the PR